deploy-registry.yaml

```diff
@@ -0,0 +1,7 @@
+- hosts: registry
+  roles:
+    - common-no-vlan
```

Comment:
This role is useless at the moment, only needed for ssh keys, maybe in the future it will be of more use.